community_supplied/Security/get-dev-key-state.rule

```diff
@@ -0,0 +1,89 @@
+/*
+ * This rule checks if development keys have been left on the system.
+ * 
+ * If development keys were present, this rule will report the results
+ * of the attempted revocation as Successful or a Failure.
+ * This rule relies on a file present in 19.3R1 and further.
+ */
+healthbot {
+    topic security {
+        rule get-dev-key-status {
```

Comment:
Thanks for contributing HB rules.
If possible, Please add rule "description" and "synopsis"